include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -6071,6 +6270,117 @@ FEEvaluationBase<dim, n_components_, Number, is_face, VectorizedArrayType>::
                 " selectively enabling this function via std::enable_if!");
 
   Tensor<2, dim, VectorizedArrayType> grad;
+
+  if (this->data->element_type ==
+      internal::MatrixFreeFunctions::ElementType::tensor_nedelec)
+    {
+      Tensor<1, (dim == 2) ? 1 : dim, VectorizedArrayType> curl_temp;
+
+      // Piola transform is required
+      AssertIndexRange(q_point, this->n_quadrature_points);
+      Assert(this->J_value != nullptr,
+             internal::ExcMatrixFreeAccessToUninitializedMappingField(
+               "update_value"));
+#  ifdef DEBUG
+      Assert(this->is_reinitialized, ExcNotInitialized());
+      this->values_quad_submitted = true;
```

Comment:
Can you please change this to
```suggestion
      this->gradients_quad_submitted = true;
```
This should fix the initial error @kkormann. There are some more errors we need to fix in our application code.